src/app/jumbotron/jumbotron.component.scss

```diff
@@ -0,0 +1,131 @@
+$breakpoint-mobile-small: 325px;
+$breakpoint-mobile: 425px;
+$breakpoint-tablet: 768px;
+$breakpoint-laptop:1024px;
+$Picton-Blue:#2AB1EA;
+$White:#FFFFFF;
+$Black : #000000;
+$Haiti : #0B132B;
+.custom-jumbo{
+    height: 433px;
+    width: 100%;
+}
+.img-div
+{
+    margin-top: 30px;
+}
+.home-img {
+	height: 315px;
+	width: 501px;
+}
+.align
+{
+    text-align: center;
+}
+.empowering-employabi {
+	height: 42px;
+	opacity: 0.8;
+	color: $Black ;
+	font-size: 36px;
+	font-weight: bold;
+    line-height: 42px;
+    display: initial;
+    font-family: Raleway;
+}
+.brainteaser{
+	height: 19px;
+	width: 530px;
+	color:$Haiti; 
+	font-family: Raleway;
+	font-size: 16px;
+	line-height: 19px;
+}
+.jumbotron p {
+    margin-bottom: 0px;
+    font-size: 16px;
+    font-weight: 100;
+    display: inline-block;
+    margin-top: 40px;
+}
+@media screen and (max-width:$breakpoint-mobile){
+    
+    .empowering-employabi
+    {
+        font-size: 20px;
+    }
+    .brainteaser{
+        width: 300px;
+    }
+}
+.oval{
+        height: 56px;	
+        width: 56px;	
+        border: 1.5px solid #2AB1EA;
+}
+.downarrow
+{
+    margin-top: 6%;
+}
+.bounce {
+    -moz-animation: bounce 3s infinite;
+    -webkit-animation: bounce 3s infinite;
```

Comment:
do not use these again
moz-animation: bounce 3s infinite;
-webkit-animation: bounce 3s infinite;

this much is enough.
animation: bounce 3s infinite;